config/literals.ts

```diff
@@ -0,0 +1,53 @@
+import { BodyParams } from "./literalTypes";
+
+export const refundRejSubject = (assignment?: string) => `Late day refund request for ${assignment} rejected`;
+export const refundRecSubject = (assignment?: string) =>  `Late day refund request for ${assignment} received`;
+export const refundAppSubject = (a?: string, t?: string) =>  `Late day request for ${a} approved: new deadline ${t}`;
+export const requestRejSubject = (assignment?: string) => `Late day request for ${assignment} rejected`;
+export const requestAppSubject = (a?: string, t?: string) => `Late day request for ${a} approved: new deadline ${t}`;
+
+export const refBeyondBody = ({assignment: a, oldDeadline: t}: Partial<BodyParams>) => [
+  `It is too late to request the refund for ${a}.`,
+  `The request should have been made by ${t}.`,
+  `Please check the rules in the syllabus.`,
+];
+
+export const refUnleftBody = ({assignment: a, oldDeadline: t}: Partial<BodyParams>) => [
+  `You didn't use any late days for ${a}.`,
+  `The original deadline for ${a} is ${t}.`,
+];
+
+export const refRecBody = ({noOfDays: d}: Partial<BodyParams>) => [
```

Comment:
```suggestion
export const refRecBody = ({numOfDays: d}: Partial<BodyParams>) => [
```
Likewise other occurrences of `noOf`.